angular-standalone/package.json

```diff
@@ -7,36 +7,36 @@
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
     "test": "ng test",
-    "cypress:run": "cypress run --config video=false --component"
+    "cypress:run": "cypress run --component"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^16.0.5",
-    "@angular/common": "^16.0.5",
-    "@angular/compiler": "^16.0.5",
-    "@angular/core": "^16.0.5",
-    "@angular/forms": "^16.0.5",
-    "@angular/platform-browser": "^16.0.5",
-    "@angular/platform-browser-dynamic": "^16.0.5",
-    "@angular/router": "^16.0.5",
-    "rxjs": "~7.5.0",
+    "@angular/animations": "^18.0.0",
+    "@angular/common": "^18.0.0",
+    "@angular/compiler": "^18.0.0",
+    "@angular/core": "^18.0.0",
+    "@angular/forms": "^18.0.0",
+    "@angular/platform-browser": "^18.0.0",
+    "@angular/platform-browser-dynamic": "^18.0.0",
+    "@angular/router": "^18.0.0",
+    "msw": "^2.3.1",
+    "rxjs": "~7.8.0",
     "tslib": "^2.3.0",
-    "zone.js": "~0.13.0"
+    "zone.js": "~0.14.3"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^16.0.5",
-    "@angular/cli": "~16.0.5",
-    "@angular/compiler-cli": "^16.0.5",
-    "@cypress/mount-utils": "^2.0.0",
-    "@types/jasmine": "~4.0.0",
-    "cypress": "^12.14.0",
-    "jasmine-core": "~4.1.0",
-    "karma": "~6.3.0",
-    "karma-chrome-launcher": "~3.1.0",
+    "@angular-devkit/build-angular": "^18.0.2",
+    "@angular/cli": "^18.0.2",
+    "@angular/compiler-cli": "^18.0.0",
+    "@types/jasmine": "~5.1.0",
+    "cypress": "https://cdn.cypress.io/beta/npm/13.12.0/darwin-arm64/angular-signals-ct-harness-c6cba8b03a7d7d098cdb469919eb674d40277604/cypress.tgz",
```

Comment:
We should try to remember to go and change this after it's released

Reply:
I wont merge it in until signals is released. Then I will update the package.json and merge it in